numba_dpex/core/utils/kernel_launcher.py

```diff
@@ -86,9 +117,15 @@ def __init__(
         """
         self.context = context
         self.builder = builder
-        self.rtctx = DpexRTContext(self.context)
         self.arguments = _KernelLaunchIRArguments()
+        self.cached_arguments = _KernelLaunchIRCachedArguments()
         self.kernel_dmm = kernel_dmm
+        self._cleanups = []
+
+    @cached_property
+    def dpexrt(self):
+        """Dpex runtime context."""
+        return DpexRTContext(self.context)
```

Comment:
Please add a new line between the docstring a first statement. It is a bit hard to read the code right now.